gluecodium/src/test/resources/smoke/comments/input/CommentsLambda.lime

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2016-2019 HERE Europe B.V.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+package smoke
+
+class LambdaComments {
+
+  // The first line of the doc.
+  // @param[p0] The first input parameter
+  lambda WithNoNamedParameters = (String) -> String
+
+  // The first line of the doc.
+  lambda WithNoDocsForParameters = (String) -> String
+
+  // The first line of the doc.
+  // @param[inputParameter] The first input parameter. The second sentence of the first input parameter.
+  // @return The string.
+  lambda WithNamedParameters = (inputParameter: String) -> String
+
+  // The first line of the doc.
+  // @param[p0] The first input parameter.
+  // @return The string.
+  lambda MixedDocNameParameters = (inputParameter: String, secondInputParameter: String) -> String
```

Comment:
Can we prevent such mixed usage?

Reply:
It is aligned with the behavior for the doc functions: if the parameter name in the doc does not match with one in the lambda definition, then this parameter will not be included in the documentation.